db/migrate/20220802014549_disable_chat_uploads_if_secure_media_enabled.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class DisableChatUploadsIfSecureMediaEnabled < ActiveRecord::Migration[7.0]
+  ##
+  # At this point in time, secure media is not compatible with discourse-chat,
+  # so if it is enabled then chat uploads must be disabled to avoid undesirable
+  # behaviour.
+  #
+  # The env var DISCOURSE_ALLOW_UNSECURE_CHAT_UPLOADS can be set to keep
+  # it enabled, but this is strongly advised against.
```

Comment:
I don't think the ENV should be the concern of a migration. Given that most people will not even have the chance to configure the ENV before the migration is run, I think we can just remove the ENV and manually update in places where it is necessary?

Reply:
Irrelevant now I have changed to GlobalSetting, also migration is fine IMO we want to automate things like this not do manual busywork. This is not really intended for average users, neither is secure_media in general really